### ISSUE TITLE
Check ENR attnets field

### ISSUE THREAD

**Issue by jrhea** [lsankar4033/stethoscope#18]
Prysm was updating the ENR with validator committee assignments.  This resulted in frequent ENR updates and leaks information about what validators a node is tied to:

Issue: Noticed high seq no’s in the topaz testnet due to frequent attnets field updates - https://github.com/ethereum/eth2.0-specs/issues/1774

Fix: Add note to clarify when to update ENR record attnet - https://github.com/ethereum/eth2.0-specs/pull/1775

Possible tests could be:
- make sure the attnet field isn't changes each time a node's validator committee assignment is modified.  Only persistent committee subscriptions should be advertised.
- If a node hosts N validators then it should subscribe to N persistent committees
- check the distribution of persistent subnet subscriptions?

If it helps prkl could be used to help inspect these fields for you.  lmk, if you need any mods, or if you would like to use it as a library. 🤷‍♂️

**Comment by jrhea** [comment]
Another implementation bug i found that is related to attnets:

Issue: Noticed anomalies in the distribution subnet subscriptions stored in enr records - https://github.com/prysmaticlabs/prysm/issues/5696

Fix: https://github.com/prysmaticlabs/prysm/pull/5734

**Comment by lsankar4033** [comment]
Definitely, testing that the proper attnets exist in the ENR over epochs should be straightforward to test
